Reduce size reducing a message
    
Easy way to reduce code but actually I found the message is
readable at the same way as previous.

Signed-off-by: Frediano Ziglio <frediano.ziglio@citrix.com>

diff --git a/mbr/gptmbr.S b/mbr/gptmbr.S
index f73b472..b0c11b9 100644
--- a/mbr/gptmbr.S
+++ b/mbr/gptmbr.S
@@ -289,7 +289,7 @@ inc64:
 
 disk_error:
 	call	error
-	.ascii	"Disk error on boot\r\n"
+	.ascii	"Disk error\r\n"
 
 /*
  * Print error messages.  This is invoked with "call", with the
